ragstack/colbert/token_embedding.py

```diff
@@ -12,13 +12,11 @@ class PerTokenEmbeddings:
 
     def __init__(
         self,
-        id: int,
-        part: int,
-        parent_id: uuid.UUID = None,
-        title: str = "",
+        id: int,  # id is the embedding id
+        part: int,  # part is the part id of the passage
+        title: str = "",  # title is the title of the passage
```

Comment:
i think we should drop `title` completely from this code. If someone wants to store the name of the file or a document `title`, they will do it in the metadata.